spec/command/apk_analyzer_spec.rb

```diff
@@ -55,6 +56,16 @@ module Apkstats::Command
       it "target_sdk should return target sdk" do
         expect(command.target_sdk(apk_base)).to eq("28")
       end
+
+      it "reference_count should return reference count" do
+        expect(command.reference_count(apk_base)).to eq(15_720.to_s)
```

Comment:
These new methods return `Fixnum`, right? so `to_s` is not required for these specs.

Reply:
Ok, removing redundant cast